src/fe-gtk/chanlist.c

```diff
@@ -310,8 +310,8 @@ chanlist_do_refresh (server *serv)
 	else
 	{
 		/* download all, filter minusers locally only */
-		serv->p_list_channels (serv, "", 1);
-		serv->gui->chanlist_minusers_downloaded = 1;
+		serv->p_list_channels (serv, "", 0);
```

Comment:
The idea seems fine but I'm not sure about this line. I never want us to actually request that many potential channels unless the *user* explicitly does so.

Reply:
That's fine. My IRC network is some kind of bridged network, so it works for me, but I understand it probably won't work for real IRC networks with tons of channels.